go/flags/endtoend/vtgate.txt

```diff
@@ -85,7 +85,7 @@ Usage of vtgate:
       --mysql_auth_server_impl string                                    Which auth server implementation to use. Options: none, ldap, clientcert, static, vault. (default "static")
       --mysql_auth_server_static_file string                             JSON File to read the users/passwords from.
       --mysql_auth_server_static_string string                           JSON representation of the users/passwords config.
-      --mysql_auth_static_reload_interval duration                       Ticker to reload credentials (default 0s)
+      --mysql_auth_static_reload_interval duration                       Ticker to reload credentials
```

Comment:
Did we lose printing out the default value?

Reply:
`pflag` only prints the default value if it differs from the zero value for that type. Since we have a duration, the default matches the zero value, so nothing gets printed 

I can look into if it's possible to have it print anyway if we're not okay with this behavior.

Reply:
Should be fine, I guess.